what4-transition-system/what4-transition-system.cabal

```diff
@@ -28,7 +27,6 @@ library
   import:           dependencies
   exposed-modules:
     What4.TransitionSystem
```

Comment:
There's another reference to Sally in `What4.TransitionSystem` that is somewhat peculiar:

https://github.com/GaloisInc/what4/blob/ea717ac94a186b5ee18f138b71d8b4b4b2f00955/what4-transition-system/src/What4/TransitionSystem.hs#L83-L87

Is this function still needed now that the Sally-specific bits have been moved out? Or, if is still needed, perhaps the comment should be revised to reflect that it does more than Sally-specific things?

Reply:
Oh yeah, this will still be needed for any output language, but the comment could be less sally-specific.